README.md

```diff
@@ -1,19 +1,31 @@
 # Pipeline Service
 
-Pipeline Service provides a SaaS for pipelines. It leverages Kubernetes, Tekton, Tekton Triggers, Tekton Results, Pipelines as Code together with kcp.
+Pipeline Service provides a SaaS for pipelines. It leverages:
+
+- kcp for serving the API and acting as control plane
+- Kubernetes / OpenShift for the compute
+- Tekton Pipelines, Results and friends for the core of the service
+- OpenShift GitOps / Argo CD, Pipelines as Code for managing the infrastructure
```

Comment:
Isn't Pipelines as Code also use for the core service? My understanding was that it was providing the service to enable hooks in git repos (with the help of hcg).

Reply:
You are correct. We use it and provide it. There is a bit of the chicken and egg story.

 I was purposely vague on the core of the service as the list is not exhaustive and may further extend, As you know we are adding Tekton chain for instance. One can understand Pipelines as Code as part of Tekton and friends.
 On the other side I wanted to emphasize that we are not only putting components together but also providing the way to manage a cohesive platform. Therefore we leverage GitOps and Pipelines as Code.
 
 I am open to proposals for a better way of formulating it. It has however to be concise. In ~1mn the reader should get a clue of the value proposal of the project and the approach taken.